ex03.ex

```diff
@@ -55,8 +55,15 @@ defmodule Ex03 do
 
   """
 
-  def odd_even . . . "your code"
+  defp odd_or_even_listOfAtoms(i) do
```

Comment:
## -1

Nice layout, and nice factoring this out.

However, as the assignment is about pattern matching, I'd rather you didn't use `cond`:

~~~ elixir
  def odd_even([]), do: []
  def odd_even([ h | t ]), do: [ atom_for(h) | odd_even(t) ]

  defp atom_for(n) when Integer.is_odd(n),  do:  :odd
  defp atom_for(n) when Integer.is_even(n), do:  :even
~~~